02_iCommands_Handson_User-Training.md

```diff
@@ -136,6 +136,14 @@ irm -r test1
 
 **Note**: The irmtrash command should be used to delete data-objects in the trash collection.
 
+In bash, you can print the contents of a file with the `cat`-command.  
+In iRODS, we can do the same with the command `istream` and the option 'read'.
+The following would print the contents of the file test1 to the terminal:
+
```

Comment:
This is not included in the official iRODS document yet. But we can put it here, a useful command!